src/pybind/mgr/rook/ci/cluster-specs/cluster-on-pvc-minikube.yaml

```diff
@@ -181,6 +181,9 @@ spec:
     managePodBudgets: true
     osdMaintenanceTimeout: 30
     pgHealthCheckTimeout: 0
+  cephConfig:
+    global:
+      mon_warn_on_pool_no_redundancy: "false"
```

Comment:
Suppressing this warning means we're using replica 1 pools. Shall we also change the .mgr pool below to use replica 1? In the CI replica 1 is typically perfectly fine.

Reply:
@travisn fixed :+1: